recipes/xsdba/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "xsdba" %}
+{% set version = "0.2.0" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/xsdba-{{ version }}.tar.gz
+  sha256: 589d8886f27aea66085ffbbcdd395d911095d2451fd76dc3eeea1c86c0dc4184
+
+build:
+  entry_points:
+    - xsdba = xsdba.cli:app
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - flit-core >=3.9,<4
+    - pip >=24.2.0
+  run:
+    - python >={{ python_min }}
+    - boltons >=20.1
+    - bottleneck >=1.3.1
+    - cf_xarray >=0.10.0
+    - cftime >=1.4.1
+    - dask-core >=2024.8.1
+    - h5netcdf >=1.3.0
+    - jsonpickle >=3.1.0
+    - numba >=0.54.1
+    - numpy >=1.23.0,<2.0
+    - pint >=0.24.3
+    - scipy >=1.9.0
+    - statsmodels >=0.14.2
+    - typer >=0.12.3
+    - xarray >=2023.11.0
+    - xclim >=0.53.2,<0.55 # temporary until xsdba v3.0
+    - yamale >=5.0.0
+
+test:
+  imports:
+    - xsdba
+  commands:
```

Comment:
Can we add a test for the cli here:
https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#test-commands

Reply:
For sure. The CLI is not yet implemented, but running `$ xsdba` will display some default messages. Will add.